### PR TITLE
Try: Graduate all Gutenberg icons to the icon font.

### PR DESCRIPTION
**Pull request by jasmussen**
Helps fix https://github.com/WordPress/gutenberg/pull/20003.

Back in the day when we moved from a manual process, to a build script for Dashicons, we put Gutenberg icons in a separate folder that was not included in the icon font. The context for that started in [a meeting](https://wordpress.slack.com/archives/C03EESJAW/p1498836928887730) — this meeting outlined the process of creating a trac ticket for every icon created and approved separately. Some of that process is [outlined here](https://wordpress.slack.com/archives/C03EESJAW/p1498838219357464). In order to stay nimble and allow us to move fast with creating Gutenberg icons, it was [suggested](https://wordpress.slack.com/archives/C03EESJAW/p1498838530476355) to put Gutenberg icons in a separate folder that is not included in the icon font, landing only in the React component.

But the icons have been there for a bit now, and in effort to fix #20003, these icons now need to graduate. Because of their time in the editor, it feels fair to review them all in a single ticket.

Here's the icon font, before:

before

Here's the icon font, after:

after

Here's a screenshot of just the new icons added:

Screenshot 2020-02-21 at 09 50 10

dashicons.ttf before: 44.956 bytes
dashicons.ttf after: 49.156 bytes

Diff: the new icons make the icon font 4.200 bytes larger. That's 0,0042 mb.

### CODE REVIEW

**Comment by youknowriad** [comment]
Thanks for the help here @jasmussen 
Not sure who's responsible for this repo and could give us the benediction to move forward.

File: codepoints.json [hunk 1]
```diff
@@ -241,7 +241,6 @@
     "trash": 61826,
     "heading": 61710,
     "insert": 61711,
-    "saved": 61715,
```
Comment (jasmussen) [inline codepoints.json]: This was stored in the codepoints file, but not used as far as I can tell. Each name in this code point is supposed to point to a file, i.e. "saved" means it was pointing to "saved.svg" in the sources/svg folder, but no such file ships in the current version. That file is stored in the sources/svg/gutenberg folder, which in this PR has been removed.

**Comment by jasmussen** [comment]
I would suggest anyone on this list, https://github.com/WordPress/dashicons/graphs/contributors, can approve this PR. 

But at some point a trac ticket will need to be created to copy the icon font into the core itself, and perhaps the design approval could be part of getting that one merged.

**Comment by jasmussen** [comment]
Thank you Stephen!

**Pull request merged**